### PR TITLE
Update prize table

### PR DESCRIPTION
**Pull request by jreis13**
Why:
* Need to add fields to the prize table.

How:
* Replace `is_merch` for `type`
* Add `description`
* Add `identifier`

### CODE REVIEW

**Review by joaojusto — CHANGES_REQUESTED** [review]
Changing migrations is not recommended. This will break the database on the other computers and had we some production database deployed it would break it as well.
I would recommend you to generate a new migration to change the prize table accordingly

**Comment by jreis13** [comment]
> Changing migrations is not recommended. This will break the database on the other computers and had we some production database deployed it would break it as well.
> I would recommend you to generate a new migration to change the prize table accordingly

Understood, fixed it now.

File: app/controllers/prizes_controller.rb [hunk 1]
```diff
@@ -23,6 +23,6 @@ def win_prize
   private
 
   def prize_params
-    params.require(:prize).permit(:name, :percentage, :image, :is_merch)
+    params.require(:prize).permit(:name, :percentage, :image, :prize_type, :description, :identifier)
```
Comment (mirhamasala) [inline app/controllers/prizes_controller.rb:26]: Shouldn't this be `type` now?

File: app/models/prize.rb [hunk 2]
```diff
@@ -4,5 +4,8 @@ class Prize < ApplicationRecord
   validates :name, presence: true
   validates :percentage, presence: true
   validates :image, presence: true
+  validates :prize_type, presence: true
```
Comment (mirhamasala) [inline app/models/prize.rb:7]: Same as above

File: db/seeds.rb [hunk 3]
```diff
@@ -9,6 +9,6 @@
 # bin/rails c    OR   rake db:reset AND bin/rails db:seed
 # rake db:drop db:create db:migrate db:seed
 5.times do |prize|
-  Prize.create(name: "Tshirt #{prize + 1}", percentage: 10, image: "assets/svgs/prizes/t-shirt.svg", is_merch: true)
-  Prize.create!(name: "NFT #{prize + 1}", percentage: 10, image: "assets/svgs/prizes/nft.svg")
+  Prize.create(name: "Tshirt #{prize + 1}", percentage: 10, image: "assets/svgs/prizes/t-shirt.svg", prize_type: 'Merch', description: 'This is a merch item', identifier: 'merch')
+  Prize.create!(name: "NFT #{prize + 1}", percentage: 10, image: "assets/svgs/prizes/nft.svg", prize_type: 'NFT', description: 'This is a NFT item', identifier: 'NFT')
```
Comment (mirhamasala) [inline db/seeds.rb:13]: Same as above

File: app/controllers/prizes_controller.rb (hunk 1 above)
Comment (jreis13) [reply app/controllers/prizes_controller.rb:26]: Was getting an error and figured that type is a reserved keyword I believe, used `prize_type` instead in this case.

Comment (mirhamasala) [reply app/controllers/prizes_controller.rb:26]: And it's okay to have `type` in the migration?

Comment (mirhamasala) [reply app/controllers/prizes_controller.rb:26]: Haven't done Rails for a while. 😅 Did you have to roll back? And create a new migration?

**Comment by mirhamasala** [comment]
Let's have @DavideSilva or @zediogoviana have the final check on this one before merging!

**Review by DavideSilva — APPROVED** [review]
Looks okay :shipit:

**Review by mirhamasala — APPROVED** [review]

**Pull request merged**